.github/workflows/main.yml

```
@@ -1,5 +1,5 @@
 ---
-name: Tests
+name: Tests & Type checking
```

NIP: or generally `CI`:

```suggestion
name: CI
```